src/encord_active/lib/common/iterator.py

```diff
@@ -79,9 +81,15 @@ def iterate(self, desc: str = "") -> Generator[Tuple[dict, Optional[Path]], None
                     self.du_hash = data_unit["data_hash"]
                     self.frame = int(data_unit["data_sequence"])
                     try:
-                        yield data_unit, next(
-                            self.project_file_structure.label_row_structure(label_hash).iter_data_unit(self.du_hash)
-                        ).path
+                        img_path = next(
+                            self.project_file_structure.label_row_structure(label_hash).iter_data_unit(self.du_hash),
+                            None,
+                        )
+                        if img_path is not None:
+                            image = Image.open(img_path.path)
```

Comment:
perhaps a try/except here if images are, e.g., only half way downloaded or corrupted in some other way.